script/test/utils.sh

```diff
@@ -160,6 +160,13 @@ version = 2
 EOF
 fi
 
+if [ ${IS_SYSTEMD}  == true ];then
```

Comment:
May be use `IS_SYSTEMD_CGROUP` so we dont get confused.

Reply:
use CGROUP_DRIVER var.